website/docs: revisit endpoint docs the nth (#19116)
goauthentik/authentik: website/docs/endpoint-devices/authentik-agent/agent-deployment/windows.md

    - **Device group _(optional)_**: select a device access group for the device to be added to after completing enrollment
    - **Expiring _(optional)_**: set whether or not the enrollment token will expire
5. Click **Create**.
6. _(Optional)_ Click the **Copy** icon in the **Actions** column to copy the enrollment token. This value will be required if [enabling a device for device compliance](#enable-device-compliance-and-local-device-login).

## Install the authentik Agent on Windows

:::info Automated deployment is recommended

> It's recommended to deploy the Agent via [MDM or automatiation tools](./automated.mdx) rather than manually.

Author: ```suggestion
It's recommended to deploy the Agent via [MDM or automatiation tools](./automated.mdx) instead of manually configuring it.
```
